src/pages/ThorChainLP/components/AddLiquitity/AddLiquidityInput.tsx

```diff
@@ -196,17 +196,39 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
   )
 
   const isSnapInstalled = useIsSnapInstalled()
-  const supportsRune =
-    walletSupportsChain({ chainId: thorchainChainId, wallet, isSnapInstalled }) &&
-    thorchainAccountIds.length > 0
-  const supportsAsset =
-    poolAsset &&
-    walletSupportsChain({
-      chainId: fromAssetId(poolAsset.assetId).chainId,
-      wallet,
-      isSnapInstalled,
-    }) &&
-    poolAssetAccountIds.length > 0
+  const supportsRune = useMemo(
```

Comment:
Same thought as the previous PR about naming this `walletSupportsRune`.

Reply:
https://github.com/shapeshift/web/pull/6152/commits/db520b9bd742eb1eacfe47339cf7dc8064811866